plugin/store/store.go

```diff
@@ -113,7 +113,7 @@ func (ps *PluginStore) getAllByCap(capability string) []CompatPlugin {
 
 	result := make([]CompatPlugin, 0, 1)
 	for _, p := range ps.plugins {
-		if _, err := p.FilterByCap(capability); err == nil {
+		if _, err := p.FilterByCap(capability); err == nil && p.IsEnabled() {
```

Comment:
@anusha-ragunathan I could not change this in the caller function because I have only access to CompatPlugin interface values, and not *Plugin objects.